Examples/Cocoa/Swift/Sources/Swifty/Board.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Board.swift
+//  Autogenerated by plank
+//
+//  DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+//  @generated
+//
+
+import Foundation
+
+public struct Board: Codable {
+    public var name: String
+    public var identifier: String
+    public var image: Image
+    public var counts: [String: Int] /* Integer */
+    public var createdAt: Date
```

Comment:
How is date parsing handled here in Codable? We had to use a transformer to translate `String -> Date`

Reply:
Basically it's up to the coder: https://littlebitesofcocoa.com/316-codable-dates.

Debating on whether the generator should define a wrapper API for a JSON coder, or just leave it up to the developer — since coding is pretty app specific.